app/main/api/util/auth/AzureAdTokenProvider.kt

```diff
@@ -31,17 +41,6 @@ class AzureAdTokenProvider(
             """.asUrlPart()
         }
 
-    suspend fun getOnBehalfOfToken(scope: String, accessToken: String) =
```

Comment:
Denne vil vi nok trenge fremover. Greit å fjerne for nå, men den vil nok komme inn igjen senere.